### PR TITLE
Add plugin entry point for integer fields.

### PR DESCRIPTION
**Pull request by defrager**
Extend plugin interface with AnalyzeIntegerField method to support annotations for integer fields.

### CODE REVIEW

File: kythe/cxx/indexer/textproto/plugin.h [hunk 1]
```diff
@@ -94,6 +94,15 @@ class Plugin {
       const google::protobuf::FieldDescriptor& field,
       const std::vector<StringToken>& tokens) = 0;
 
+  // Optional entrypoint for integer fields. Plugin may override it to add
+  // additional nodes for integer fields.
+  virtual absl::Status AnalyzeIntegerField(PluginApi* api,
+                                           const proto::VName& file_vname,
+                                           const proto2::FieldDescriptor& field,
```
Comment (shahms) [inline kythe/cxx/indexer/textproto/plugin.h:101]: externally these are in the `google::protobuf` namespace

Comment (defrager) [reply kythe/cxx/indexer/textproto/plugin.h]: Thanks! Missed it. Updated.

https://buildkite.com/bazel/kythe/builds/6875

**Comment by shahms** [comment]
Can you run clang-format on  `kythe/cxx/indexer/textproto/analyzer.cc`?

```
diff --git a/kythe/cxx/indexer/textproto/analyzer.cc b/kythe/cxx/indexer/textproto/analyzer.cc
index ce49b0fd0..8cf52bcc8 100644
--- a/kythe/cxx/indexer/textproto/analyzer.cc
+++ b/kythe/cxx/indexer/textproto/analyzer.cc
@@ -635,7 +635,7 @@ absl::Status TextprotoAnalyzer::AnalyzeIntegerValue(
                                 field.full_name());
     }
     input = input.substr(match.size());
-      
+
     for (auto& p : plugins_) {
       auto s = p->AnalyzeIntegerField(this, file_vname, field, match);
       if (!s.ok()) {
diff --git a/kythe/cxx/indexer/textproto/plugin.h b/kythe/cxx/indexer/textproto/plugin.h
index 431ade689..959046707 100644
--- a/kythe/cxx/indexer/textproto/plugin.h
+++ b/kythe/cxx/indexer/textproto/plugin.h
@@ -97,8 +97,7 @@ class Plugin {
   // Optional entrypoint for integer fields. Plugin may override it to add
   // additional nodes for integer fields.
   virtual absl::Status AnalyzeIntegerField(
-      PluginApi* api,
-      const proto::VName& file_vname,
+      PluginApi* api, const proto::VName& file_vname,
       const google::protobuf::FieldDescriptor& field,
       absl::string_view field_value) {
     return absl::OkStatus();
```

**Comment by defrager** [comment]
Diff applied. Thanks!

**Review by shahms — APPROVED** [review]

**Pull request merged**